recipe/meta.yaml

```diff
@@ -11,18 +11,17 @@ source:
 
 build:
   number: 1
-  msvc_compiler: 14.0  # [win]
-  # NOTE: Ninja is agnostic to the Python version, so there is no point in
-  # building it on every python version. On the other hand, Windows CI fails
-  # to build Ninja using Python 3.5, so we stick to Python 2.7 for all
-  # platforms to be consistent.
-  skip: True  # [not py27]
+  skip: true  # [win and vc!=14]
+  ignore_run_exports:
+    - vc
 
 requirements:
   build:
+    - {{ compiler("cxx") }}
     - python
+
   run:
-    - vc 14  # [win]
+    - vs2015_runtime  # [win]
```

Comment:
Do we still need to list `vs2015_runtime` given we are using `{{ compiler("cxx") }}`, @msarahan?

Reply:
It still needs to be listed, but IMO, it shouldn't be needed. Only `vc` package is added as a `run_exports`. https://github.com/AnacondaRecipes/aggregate/blob/fe73fb12a1755d6e58156ee6bf3e72061595a7fc/vs2008/meta.yaml#L20. I think both `vc` and `vs{{ vsyear }}_runtime` should `run_exports`. So we can turn off `vc` when needed in cases like this, and I can't think of a case where the runtime package run_export needs to be turned off.